assets/js/pages/Users/UsersPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { listUsers, deleteUser } from '@lib/api/users';
+
+import { toast } from 'react-hot-toast';
+
+import Users from './Users';
+
+const SUCCESS_DELETE_MESSAGE = 'User deleted successfully';
+const ERROR_LOADING_MESSAGE = 'An error occurred during loading users';
+const ERROR_DELETING_MESSAGE = 'An error occurred during deleting user';
```

Comment:
"while loading users" or "during the loading of users"
"while deleting user" or "during the deletion of user"